src/client/wire_protocol/operations.rs

```diff
@@ -53,6 +53,13 @@ pub enum Message {
         query: BsonDocument,
         return_field_selector: Option<BsonDocument>,
     },
+    OpGetMore {
+        header: Header,
+        // ZERO goes here
```

Comment:
I'd prefer a comment that's more descriptive (like your reply that makes it clear that the ZERO field is for the wire protocol).